configuration/uart.md

```diff
@@ -1,99 +1,100 @@
 # UART configuration
 
-There are two types of UART available on the Raspberry Pi -  [PL011](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.ddi0183g/index.html) and mini UART. The PL011 is a capable, broadly 16550-compatible UART, while the mini UART has a reduced feature set.
+A [UART](https://en.wikipedia.org/wiki/Universal_asynchronous_receiver-transmitter) is a serial communication device which uses one or more signal wires and a common ground to transmit and receive data between two devices. When computer people talk about 'serial ports', they usually mean UARTs. People from DOS/Windows background may know these as 'COM ports'.
 
-All UARTs on the Raspberry Pi are 3.3V only - damage will occur if they are connected to 5V systems. An adaptor can be used to connect to 5V systems. Alternatively, low-cost USB to 3.3V serial adaptors are available from various third parties.
+## Hardware
 
-## Pi Zero, 1, 2 and 3 - two UARTs
+On all Raspberry Pi computers except microcontrollers, there are at least two UARTs built into the [SoC](https://en.wikipedia.org/wiki/System_on_a_chip). The Raspberry Pi 4 family has six UARTs.
```

Comment:
The Raspberry Pi Pico is a microcontroller, and so doesn't fall into the "Raspberry Pi computers" category :wink: 
**All** the Pico documentation is at https://www.raspberrypi.org/documentation/rp2040/getting-started/ and the rest of the documentation repo only covers "Raspberry Pi computers" without any mention of the Raspberry Pi microcontroller.

(fun fact: the Raspberry Pi Pico also has 2 UARTs :laughing: )

Reply:
This. There should be no mention of Pico in the mainline documentation. That split happens (will happen) further up the documentation tree.

Reply:
Great, thanks for the context. This is adjusted in the next rev.